apps/microsoft-teams/frontend/src/context/ChannelProvider.tsx

```diff
@@ -4,6 +4,8 @@ import { TeamsChannel } from '@customTypes/configPage';
 
 interface ChannelContextValue {
   channels: TeamsChannel[];
+  loading: boolean;
```

Comment:
Happy to make these more consistent, I think I specified it in the ContentType provider because of the additional config link attribute (which I originally thought might also have a loading state until I looked at the code!). Will adjust 👍